infection.json

```diff
@@ -0,0 +1,24 @@
+{
+  "ignoreMsiWithNoMutations": true,
+  "logs": {
+    "text": ".build/infection/infection-log.txt"
+  },
+  "minCoveredMsi": 100,
+  "minMsi": 100,
+  "phpUnit": {
+    "configDir": "."
+  },
+  "source": {
+    "directories": [
+      "src"
+    ],
+    "excludes": [
+      "ExampleBench.php",
+      "ExampleTest.php",
+      "Helper.php",
+      "ValueCanNotBeBlankTest.php",
+      "ValueCanNotBeEmptyTest.php"
+    ]
```

Comment:
When you declare test code in the same directory as production code, managing exclusions for `infection/infection` in `infection.json` is a lot of work.